screwdriver.yaml

```diff
@@ -2,20 +2,28 @@ workflow:
     - publish
 
 shared:
-    image: node:6
+    image: centos
 
 jobs:
     main:
         steps:
-            - install: npm install
-            - test: npm test
-
+            # To verify if the hyperctl-download script works
+            - hyperctl-download: bash -xe scripts/hyperctl-download.sh
```

Comment:
Not a blocker: add a step that explicitly checks the binary exists after download